README.md

```diff
@@ -6,8 +6,15 @@
 Describe why your topic is important and what you want to share with your audience
 
 ## Requirements
-Do not forget to indicate Python version and any other tools
-+ add requirements.txt or conda.yml or docker image or Binder/Google Collab link
+PyLadies encourage the usage of [uv](https://astral.sh/blog/uv) for dependency management. You can get uv by running the following command in your shell:
```

Comment:
I would rather add uv and ruff pre-commit here as a hard requirement with direct links to how to install them from the source docs.
And move the rest of the info (lines 13-17 to the usage)

Reply:
I adjusted the wording here and added additional links.